active_learning_multiannotator/active_learning_single_annotator.ipynb

```diff
@@ -0,0 +1,461 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Active Learning for caltech256 using Cleanlab"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/cleanlab/examples/blob/master/active_learning_multiannotator/active_learning.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates a practical approach to active learning for training classification models with cleanlab. In active learning, we aim to construct a labeled dataset by collecting the fewest labels that still allow us to train an accurate classifier model. Here we assume data labeling is done in **batches**, and between these data labeling rounds, we retrain our classifier to decide what examples (i.e. datapoints) to label next round. We consider labeling a batch of examples per round and are limited to a single annotation per example.\n",
+    "\n",
+    "cleanlab provides an active learning score quantifying how desirable it is to collect an additional label for every possible unlabeled example\n",
+    "\n",
+    "This notebook demonstrates how to compute these easily for use in sequential active learning, showing how a classification model iteratively improves after labeling more examples for multiple rounds.\n",
+    "This notebook implements the following steps:\n",
+    "\n",
+    "1. Establish some already labeled data. Use this data to train a classifier model and then obtain out-of-sample predicted probabilities for each labeled and unlabeled example.\n",
+    "2. Compute active learning scores for every example, which estimate our current confidence in knowing its true label.\n",
+    "3. Collect additional labels for the unlabeled examples with the lowest active learning scores. These are the most potentially informative examples whose true label we are least certain of. \n",
+    "4. Repeat the steps above to collect as many labels as your budget permits.\n",
+    "\n",
+    "The accuracy of the model trained on the resulting dataset will generally match that of the same model trained on a much larger set of randomly collected labels. I.e. this is the most cost-effective way to train an accurate classifier!\n",
+    "\n",
+    "In this example we use the caltch256 dataset combined with autogluon's new MultiModalPredictor architecture."
```

Comment:
```suggestion
    "In this example, we consider labeling images from the Caltech-256 dataset. We train an  image classification model using AutoML methods from the AutoGluon library."
```

Reply:
note: you have typos in the dataset name such as here.  Also the dataset name should be consistent:

Caltech-256

everywhere, not sometimes lower-case, etc.

Reply:
Also tip is to not say "new" in a notebook like this which will be online for years. Goal is to keep the notebook timeless. The biggest danger is always including links which may break in future, so always be thoughful about links